frontend/packages/knative-plugin/src/components/overview/OverviewDetailsKnativeResourcesTab.tsx

```diff
@@ -45,6 +46,18 @@ const getSidebarResources = ({ obj, ksroutes, revisions, configurations }: Overv
 };
 const OverviewDetailsKnativeResourcesTab: React.FC<OverviewDetailsResourcesTabProps> = ({
   item,
-}) => <div className="overview__sidebar-pane-body"> {getSidebarResources(item)} </div>;
+}) => (
+  <div className="overview__sidebar-pane-body">
+    {item.isOperatorBackedService && (
+      <span className="sidebar__section-owner-operator-heading">
+        Managed by operator:
+        <span className="sidebar__section-owner-reference-operator">
+          <OwnerReferences resource={item.obj} />
```

Comment:
Looks like owner references can be an array. Does this mean that every owner ref should be listed or only the one that is backed by an operator?
Reusing this component is probably ok for now but we should ok investigate.

Reply:
There could also be multiple owners backed by operators

Reply:
in that case we would need a check for the operator backed owner.
I'm not sure if there can be multiple operator backed owners.

Reply:
in that case we would need a check for the operator backed owner.
I'm not sure if there can be multiple operator backed owners.